.github/workflows/ci.yml

```diff
@@ -304,8 +305,6 @@ jobs:
           name: babel8-artifact
       - name: Extract artifacts
         run: tar -xf babel-artifact.tar; rm babel-artifact.tar
-      - name: Generate runtime helpers
-        run: make build-plugin-transform-runtime-dist
```

Comment:
This change should have been included in https://github.com/babel/babel/pull/15165.